src/components/input-field.jsx

```diff
@@ -1,6 +1,10 @@
+// @flow
 import React from 'react';
 
-const InputField = ({ captureField, name, triggers }) => <label> {name.replace(name[0], name[0].toUpperCase())}: <input type='text' name={name} id={name}
-onChange={(e) => captureField(e, triggers)}/>  <br/></label>;
+// types
+import type { Element } from 'react';
+import type { InputFieldParams, eventTargetIdValue } from '../lib/type';
+
+const InputField = ({ captureField, name, triggers }: InputFieldParams): Element<*> => <label> {name.replace(name[0], name[0].toUpperCase())}: <input type='text' name={name} id={name} onChange={(e: eventTargetIdValue) => captureField(e, triggers)}/>  <br/></label>;
```

Comment:
For this application this will work. Since this is a "normalize" function for field label, this should be a helper function. By putting this into a helper function sooner, you set the stage for scalability use later down the road.